DevelopmentProposals/reporterColumnLabels/reporterColumnLabels.md

```diff
@@ -54,14 +54,15 @@ and forcing the user to specify an annotation for each Output may not be practic
 - The TableReporter should use a default column name that is very likely to be meaningful.
 
 ### Architecture
-The selected design involves four main changes:
+The selected design involves six main changes:
 
 1. Rename "annotation" to "alias" (e.g., `AbstractInput::getAnnotation()` becomes `AbstractInput::getAlias()`).
 2. If the user provides the optional second argument when connecting an Output to an Input, store the string as the Input's alias; otherwise, the alias remains null.
-3. Create `getShortLabel()` and `getLongLabel()` methods on Input:
-  - `getShortLabel()` will be called by ConsoleReporter and will return `isNull(alias) ? output_name : alias`.
-  - `getLongLabel()` will be called by TableReporter and will return `isNull(alias) ? full_path_name : alias`.
-4. Create `setAlias()` methods analogous to the current `getAnnotation()` methods so the user can change the alias after (but not before) connecting.
+3. Create `getReportedLabel()` on Input, which will always return the alias if one is provided:
+  - `getReportedLabel()` will be called by a Reporter and will return `isNull(alias) ? full_path_name : alias`.
```

Comment:
There was no secret follow up discussion. I should have raised my concerns earlier, but they did not come to me immediately at the meeting. I thought a bit about it over the weekend. I tried to discuss it with you in person first but you demanded I update the DevProp, which I did. A 15-20min direct conversation could have established agreement to revise/or not. As the principle on this PR if you don't agree then there isn't an agreement.  I voiced my suggestions/concerns and they were rejected. I am totally fine with that.

I apologize for the disruption and I will strive to be more timely in my feedback. As a matter of team dynamics, we should all feel free to discuss any concerns we have at anytime in the dev process.  Those concerns can be addressed, squashed or postponed, but discussing it with each other should be easy and should not be reserved for weekly meetings.

Thank you for carefully reviewing the proposal and providing your feedback. @jenhicks also recommended against the revision, which I believe is your position as well. I will close it.